base-node/index.d.ts

```diff
@@ -197,6 +199,13 @@ export class BaseNode<M extends Meta = Meta> {
    */
   remoteSubprotocol: string | undefined
 
+
+  /**
+   * Headers set by remote node.
+   * By default, it is an empty object.
```

Comment:
We should have examples for all public methods

Reply:
Added